### PR TITLE
More vaults to spaces

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR takes another batch at renaming `vaults` to `spaces`.

### CODE REVIEW

File: front/lib/resources/string_ids.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ const REGION = 1; // US.
 const RESOURCES_PREFIX = {
   file: "fil",
   group: "grp",
-  vault: "vlt",
+  // TODO(2024-10-31 flav) Add new prefix for space.
```
Comment (spolu) [inline front/lib/resources/string_ids.ts:24]: I think you can just remove the TODO no? Unlikely we'll ever get to this one no?

**Review by spolu — APPROVED** [review]
LGTM

File: front/lib/resources/string_ids.ts (hunk 1 above)
Comment (spolu) [reply front/lib/resources/string_ids.ts:24]: I mean we could rewrite on the fly while we migrate maybe...

Comment (flvndvd) [reply front/lib/resources/string_ids.ts:24]: We will, I was thinking about supporting two prefixes for one resource type. I'm all in, no more vaults 🙃.

Comment (spolu) [reply front/lib/resources/string_ids.ts:24]: Until they rename it into something else :P

**Pull request merged**